src/coreclr/src/debug/di/CMakeLists.txt

```diff
@@ -79,5 +79,5 @@ endif (CLR_CMAKE_TARGET_WIN32)
 list(APPEND CORDBDI_SOURCES ${CORDBDI_SOURCES_ASM_FILE})
 
 add_library_clr(cordbdi STATIC ${CORDBDI_SOURCES})
-target_precompile_header(TARGET cordbdi HEADER stdafx.h)
+target_precompile_headers(cordbdi PRIVATE stdafx.h)
```

Comment:
The `[["stdafx.h"]]` syntax adds the literal include "stdafx.h", whereas the `stdafx.h` syntax adds an include for the stdafx.h file in the directory. Since we don't always use `CMAKE_INCLUDE_SOURCE_DIR`, I've used the `stdafx.h` syntax over the literal syntax where possible.